pcdsdevices/epics/lodcm.py

```diff
@@ -232,39 +243,19 @@ def lodcm_move(self, h1n=None, yag=None, dectris=None,
         done_statuses = []
         for state, obj in zip(states, obj):
             if state is not None:
-                status = obj.move(state)
+                status = obj.set(state)
                 done_statuses.append(status)
 
         lodcm_status = DeviceStatus(self, timeout=timeout)
         if not done_statuses:
             lodcm_status._finished(success=True)
             return lodcm_status
 
-        events = []
-        for i, status in enumerate(done_statuses):
-            event = Event()
-            events.append(event)
-            status.add_callback(self._make_mark_event(event))
+        first_status = done_statuses[0]
```

Comment:
I think you forgot to put in the `status_wait` here unless I'm missing. This will still return immediately with `wait` set to True.

Reply:
You are correct, I forgot to handle the `wait` keyword... but this was a bug before. Good catch, I'll handle this when we clean up the lodcm class.

Reply:
👍